### PR TITLE
feat: add Breadcrumbs

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
Breadcrumb component

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/Breadcrumbs/Breadcrumbs.component.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+/**
+ * Indicate the current page's location within a navigational hierarchy.
+ * @param {object} props   react props
+ * @example
+ <Breadcrumbs items={items}/>
+ */
+function Breadcrumbs(props) {
+	const theme = props.theme || {};
+	const maxItemsToDisplay = props.maxItems || 3;
+	const nbItems = props.items.length;
```
Comment (jmfrancois) [inline src/Breadcrumbs/Breadcrumbs.component.js]: raise exception if items is not defined

File: src/Breadcrumbs/Breadcrumbs.component.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	const maxItemsReached = nbItems > maxItemsToDisplay;
+	const ellipsisIndex = (nbItems - 1) - maxItemsToDisplay;
+	return (
+		<ol className={classNames('breadcrumb', theme.breadcrumb)}>
+			{props.items.map(
+				(item, index) => {
```
Comment (jmfrancois) [inline src/Breadcrumbs/Breadcrumbs.component.js]: Please refactor this arrow function in a normal function:
```function renderItem(item, index) {
....
}
...
{props.items.map(renderItem)}
```

to not have a so deep indentation level.

File: src/Breadcrumbs/Breadcrumbs.test.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+describe('Breadcrumbs', () => {
+	const items = [
+		{ text: 'Text A' },
+		{ text: 'Text B' },
+		{ text: 'Text C' },
+		{ text: 'Text D' },
+	];
+
+	it('should render items', () => {
```
Comment (acateland) [inline src/Breadcrumbs/Breadcrumbs.test.js]: test description should include that an ellipsis is intended as output

**Comment by frassinier** [comment]
[Bootstrap theme](Talend/bootstrap-theme) needs some rework.

**Pull request merged**